crates/vm/src/arch/mod.rs

```diff
@@ -1,7 +1,7 @@
 mod config;
 /// Instruction execution traits and types.
 /// Execution bus and interface.
-mod execution;
+pub mod execution;
```

Comment:
there is also `pub use execution::*`, only one of them should be `pub`?

Reply:
yep, my bad. i'll revert